### ISSUE TITLE
Ability to add a render group to the vue/no-unused-properties rule

### ISSUE THREAD

**Issue by danyadev** [vuejs/eslint-plugin-vue#1196]
**Tell us about your environment**	
* **ESLint version:** 7.2.0
* **eslint-plugin-vue version:** 7.0.0-alpha.5
* **Node version:** v12.16.3

**Please show your full configuration:**

```js
module.exports = {
  root: true,

  parserOptions: {
    sourceType: 'module'
  },

  env: {
    browser: true,
    node: true,
    es2020: true
  },

  plugins: ['import'],

  settings: {
    'import/resolver': 'webpack',
    'import/ignore': []
  },

  extends: ['plugin:vue/vue3-recommended'],

  rules: {
    // *eslint rules*
    // *import rules*
    // *other vue rules*

    'vue/no-unused-properties': ['error', {
      groups: ['props', 'setup']
    }]
  }
};
```

**What did you do?**	

```vue
<script>
import { h } from 'vue';

export default {
  props: ['content'],

  render(props) {
    return h('div', props.content);
  }
};
</script>
```

**What did you expect to happen?**
Ability to add a `render` group and avoid this error

**What actually happened?**
```
5:11  error  'content' of property found, but never used  vue/no-unused-properties
```

------------------------
When choosing the type of issue was written `Bug report` and `Rule proposal`. I chose `Bug report` because this issue is more suitable for this type.
